### ISSUE TITLE
:rtype: is removed from doc comments; :return: loses indentation

### ISSUE THREAD

**Issue by patkujawa-wf** [dadadel/pyment#9]
https://pythonhosted.org/an_example_pypi_project/sphinx.html#python-cross-referencing-syntax shows that `:rtype:` is the sphinx way to specify the type or return items, but pyment removes that tag from the docstring. It also seems to de-dent or lose the indentation of the `:return:` tag:

``` patch
# Patch generated by Pyment v0.3.0

--- a/constants.py
# Patch generated by Pyment v0.3.0

--- a/constants.py
+++ b/constants.py
@@ -122,8 +124,8 @@

     :param range: A viewer structure range
     :type range: list or tuple
-    :return: A nice representation
-    :rtype: tuple
+:return: A nice representation
+
     """
     vs = VIEWER_STRUCTURE_KEYS
     _type = range[vs.RANGE_TYPE_IDX]
```

**Comment by patkujawa-wf** [comment]
@dadadel I'm confused, now, since http://sphinx-doc.org/domains.html#info-field-lists has an example that uses `:return:`, not `:returns:`. So which is correct?

```python
.. py:function:: send_message(sender, recipient, message_body, [priority=1])

   Send a message to a recipient

   :param str sender: The person sending the message
   :param str recipient: The recipient of the message
   :param str message_body: The body of the message
   :param priority: The priority of the message, can be a number 1-5
   :type priority: integer or None
   :return: the message id
   :rtype: int
   :raises ValueError: if the message_body exceeds 160 characters
   :raises TypeError: if the message_body is not a basestring
```

**Comment by maqifrnswa** [comment]
Both are correct, return is an alias for returns:
https://github.com/sphinx-doc/sphinx/blob/2ec179044bc7dfc647731a08bc3da7d6e133c994/sphinx/domains/c.py#L69
https://github.com/sphinx-doc/sphinx/blob/5f63aac209b0d23420ddb6374673ed13dd7ec4b2/doc/ext/napoleon.rst#L110

found this while running test suites, I'll take a look at it too

**Comment by dadadel** [comment]
@maqifrnswa Thanks for your fix. I updated your test commit.

**Comment by dadadel** [comment]
The issue is fixed by the PR #28. However as @patkujawa-wf and @maqifrnswa pointed, the two syntaxes are valid, so I keep the issue opened as I consider it should be better to accept both and choose the default one via an option.
